bin/main/org/fundacionjala/at15/katas/README.md

```diff
@@ -0,0 +1,43 @@
+# Katas folder
```

Comment:
Please remove `bin/` folder and add it to `.gitignore`

Reply:
All ready done it